### PR TITLE
Remove cherrypick processor and related configuration options

### PR DESCRIPTION
**Pull request by goosemania**
closes #7869
https://pulp.plan.io/issues/7869

### CODE REVIEW

File: CHANGES/7869.removal [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Cherrypick processor is no longer available. 
+
+Configuraiton options `cherry_pick_automation` and `stable_branch` are no longer in use.
```
Comment (mdellweg) [inline CHANGES/7869.removal:3]:
```suggestion
Configuration options `cherry_pick_automation` and `stable_branch` are no longer in use.
```

File: plugin-template [hunk 2]
```diff
@@ -57,6 +55,8 @@ DEFAULT_SETTINGS = {
 
 
 DEPRECATED_FILES = [
+    ".ci/scripts/cherrypick.sh",
+    ".ci/scripts/cherrypick.sh",
```
Comment (mdellweg) [inline plugin-template:59]: Twice the same file? Maybe `.py`?
```suggestion
    ".ci/scripts/cherrypick.py",
```

**Review by mdellweg — CHANGES_REQUESTED** [review]

File: plugin-template (hunk 2 above)
Comment (fao89) [inline plugin-template:59]:
```suggestion
    ".ci/scripts/cherrypick.sh",
    ".ci/scripts/cherrypick.py",
```

**Comment by goosemania** [comment]
I tested it, it showed me 2 deprecated files :D I was satisfied :D Didn't notice that it was the same file twice ;) Thank you both.

**Review by fao89 — APPROVED** [review]

**Review by mdellweg — APPROVED** [review]

**Pull request merged**